codalab/worker/worker.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         commit_file,  # type: str
         cpuset,  # type: Set[str]
         gpuset,  # type: Set[str]
+        max_memory,  # type:Optional[int]
```

Comment:
Add a space between `:` and `Optional`

Reply:
Thanks for the review.
Space is added.
Regarding square bracket, it seems the other occurrences also use `Optional[{data type
]` style such as `dependency_manager,  # type: Optional[DependencyManager]` and `local_bundles_dir,  # type: Optional[str].

Reply:
Yes, you are right. Annotated arguments with a default value of None should use `[]`.